dm/syncer/dbconn/db.go

```diff
@@ -84,7 +85,12 @@ func (conn *DBConn) ResetConn(tctx *tcontext.Context) error {
 }
 
 // QuerySQL does one query.
-func (conn *DBConn) QuerySQL(tctx *tcontext.Context, query string, args ...interface{}) (*sql.Rows, error) {
+func (conn *DBConn) QuerySQL(
+	tctx *tcontext.Context,
+	metricProxies *metrics.Proxies,
```

Comment:
Can we put `metricsProxies` in `tctx`?

Reply:
tcontext is a common package, not only for syncer's metrics. And some functions don't have a `tctx` argument, so I put metrics outside `tctx`